react-native-libraries.json

```
@@ -12124,5 +12124,24 @@
     "expoGo": true,
     "dev": true,
     "newArchitecture": true
+  },
+  {
+    "githubUrl": "https://github.com/adobe/aepsdk-react-native",
+    "npmPkg": "@adobe/react-native-aepcore",
+    "examples": [
+      "https://github.com/adobe/aepsdk-react-native/tree/main/apps/AEPSampleApp"
+    ],
+    "ios": true,
+    "android": true,
+    "web": false,
+    "expoGo": true,
+    "windows": false,
+    "macos": false,
+    "tvos": false,
+    "visionos": false,
+    "unmaintained": false,
+    "dev": false,
+    "template": false,
+    "newArchitecture": false
```

```suggestion
    "android": true,
    "expoGo": true
```

Formatting nit, `false` flags are redundant.

---

updated - thanks! 🙌